src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/Commands.Management.CognitiveServices.csproj

```diff
@@ -0,0 +1,169 @@
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Models\PSCognitiveServicesAccount.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="CognitiveServicesAccount\GetAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\GetAzureCognitiveServicesAccountKey.cs" />
+    <Compile Include="CognitiveServicesAccount\GetAzureCognitiveServicesAccountSkus.cs" />
+    <Compile Include="CognitiveServicesAccount\NewAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\NewAzureCognitiveServicesAccountKey.cs" />
+    <Compile Include="CognitiveServicesAccount\RemoveAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\SetAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\CognitiveServicesAccountBaseCmdlet.cs" />
+    <Compile Include="CognitiveServicesAccount\CognitiveServicesManagementClient.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\AzureRM.CognitiveServices.psd1">
+      <Link>AzureRM.CognitiveServices.psd1</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="Microsoft.Azure.Commands.Management.CognitiveServices.dll-Help.psd1">
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+    </None>
+    <None Include="Microsoft.Azure.Commands.Management.CognitiveServices.format.ps1xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <Content Include="Microsoft.Azure.Commands.Management.CognitiveServices.dll-Help.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <SubType>Designer</SubType>
+    </Content>
+    <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication\Commands.Common.Authentication.csproj">
+      <Project>{d3804b64-c0d3-48f8-82ec-1f632f833c9e}</Project>
+      <Name>Commands.Common.Authentication</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.ResourceManager.Common\Commands.ResourceManager.Common.csproj">
+      <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
+      <Name>Commands.ResourceManager.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Resources\Commands.Resources\Commands.Resources.csproj">
```

Comment:
Do not create a project reference to the resource manager cmdlets in your cmdlets project.  This prevents independently versioning the generic resource manager and cognitive services cmdlets.